src/tools/schema2kotlin.ts

```diff
@@ -71,6 +73,7 @@ ${this.opts.wasm ? 'import arcs.sdk.wasm.*' : 'import arcs.core.data.RawEntity\n
       const handleName = connection.name;
       const entityType = `${particleName}_${this.upperFirst(connection.name)}`;
       const handleConcreteType = connection.type.isCollectionType() ? 'Collection' : 'Singleton';
+      const queryType = this.getQueryType(connection);
```

Comment:
This is unused?

Reply:
Temporarily, there should be some code here that generates the handle API / facade so that we can generate the query function.

It should be something like:
```
fun query(query_argument: ${queryType})
```

Happy to take it out and have an unused function instead of an unused variable if that is considered better.